src/main/java/green/conway/RleParser.java

```diff
@@ -0,0 +1,106 @@
+package green.conway;
+
+public class RleParser {
+
+    private final int xpos = 2;
+    private final int ypos = 5;
```

Comment:
This is only true if x < 10

Reply:
Even though the method separates by space?

I can't figure out why sometimes the gui works and sometimes it doesn't

Reply:
What happens when it doesn't work?

Reply:
The Gosper glider gun starts out in the right position but sometime in the next several generations mutates (by 20 it’s very clearly not right). It performs the same way in when I enter via the text field. For pi-heptomino, the first time I enter it in the text field (per running of the lifeFrame) it starts in the right position but then dies within two generations. If I clear the field and re-enter, it only starts out with the bottom two rows visible, and dies within one. (Then when I rerun the whole program, it runs correctly the first time only.) From what I can tell the only major changes I’ve made since the glider gun stopped working correctly is dealing with the resetGridViaParser() method and the timer in lifeComponent. This is a screen recording-- https://drive.google.com/file/d/1bEiFob_b1ofxP22j22zhtz2cwhnItm7R/view?usp=sharing

Reply:
Your Grid is probably not big enough.
We will address this in class.